.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+name: Prime Data Hub
+
+on:
+  push:
+    branches: [ master, production ]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set Environment Variables
+        run: |
+               if [ "$GITHUB_BASE_REF" == "production" ]
+               then
+                   echo "Deploying to the production environment."
+                   echo >> $GITHUB_ENV RESOURCE_GROUP=prime-data-hub-prod
+                   echo >> $GITHUB_ENV AZ_FXNS_DEFAULT_KEY=${{ secrets.AZ_FXNS_PROD_DEFAULT_KEY }}
+                   echo >> $GITHUB_ENV POSTGRESQL_USER=${{ secrets.POSTGRESQL_PROD_USER }}
+                   echo >> $GITHUB_ENV POSTGRESQL_PWD=${{ secrets.POSTGRESQL_PROD_PWD }}
+                   echo >> $GITHUB_ENV SFTP_CREDS=${{ secrets.SFTP_PROD_CREDS }}
+               else
+                   echo "Deploying to the test environment."
+                   echo >> $GITHUB_ENV RESOURCE_GROUP=prime-data-hub-test
+                   echo >> $GITHUB_ENV AZ_FXNS_DEFAULT_KEY=${{ secrets.AZ_FXNS_TEST_DEFAULT_KEY }}
+                   echo >> $GITHUB_ENV POSTGRESQL_USER=${{ secrets.POSTGRESQL_TEST_USER }}
+                   echo >> $GITHUB_ENV POSTGRESQL_PWD=${{ secrets.POSTGRESQL_TEST_PWD }}
+                   echo >> $GITHUB_ENV SFTP_CREDS=${{ secrets.SFTP_TEST_CREDS }}
+               fi
+               env
+
+      - name: Login to Azure CLI
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.SERVICE_PRINCIPAL_CREDS }}
+
+      - name: Update Azure Functions App
+        run: az functionapp update --resource-group ${{ env.RESOURCE_GROUP }} --name ${{ env.RESOURCE_GROUP }}
+
+      - name: Set Azure Function Default Key
+        run: |
+              az functionapp function keys set \
+              --resource-group ${{ env.RESOURCE_GROUP }} \
+              --name ${{ env.RESOURCE_GROUP }} \
+              --function-name reports \
+              --key-name default \
+              --key-value ${{ env.AZ_FXNS_DEFAULT_KEY }}
+
+      - name: Set PostgreSQL Environment Variables
+        run: | 
+              az functionapp config appsettings set \
+              --resource-group ${{ env.RESOURCE_GROUP }} \
+              --name ${{ env.RESOURCE_GROUP }} \
+              --settings POSTGRES_USER=${{ env.POSTGRESQL_USER }}@${{ env.RESOURCE_GROUP }} \
+                         POSTGRES_PASSWORD=${{ env.POSTGRESQL_PWD }} \
+                         POSTGRES_URL=jdbc:postgresql://${{ env.RESOURCE_GROUP }}.postgres.database.azure.com:5432/prime_data_hub?sslmode=require
+
+      - name: Set SFTP Environment Variables
+        run: |
+              az functionapp config appsettings set \
+              --resource-group ${{ env.RESOURCE_GROUP }} \
+              --name ${{ env.RESOURCE_GROUP }} \
+              --settings PRIME_ENVIRONMENT=test ${{ env.SFTP_CREDS }}
```

Comment:
This is a space-delimited set of creds (currently only AZ) that we can add to for new facilities.  We will likely move onto Key vault as soon as we can.